package exceptions.testthrows;

public class A {
	public void f1(){
		
	}
}
